async_service/asyncio.py

```diff
@@ -290,6 +291,16 @@ def run_child_service(
         self.run_task(child_manager.run, daemon=daemon, name=task_name)
         return child_manager
 
+    @property
+    def stats(self) -> Stats:
+        total_count = max(0, len(self._service_task_dag) - 1)
+        finished_count = min(
+            total_count, len([task for task in self._service_task_dag if task.done()])
```

Comment:
If this list is going to be big (say 1M entries), it could be problematic to calculate these on the fly. But I'm guessing there will be orders of magnitude fewer tasks, so probably something we can punt on.

```
In [1]: %timeit len([i for i in range(1000000)])
40.2 ms ± 253 µs per loop (mean ± std. dev. of 7 runs, 10 loops each)
```